lib/base-wizard.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
```

Comment:
hmpo wizard is an extension of hmpo controller and it's also our entry point to the hmpo controller.
For instance, in /routes/index.js, we require the wizard to initialise our routes with the steps and fields. At that point if no custom controller is defined in a step, then the wizard will delegate to the hmpo controller and so the base wizard is required in order for us to be able to return the correct implementation of the controller/error (base-controller/base-error) to the wizard, otherwise the wizard will circumvent our base-*'s. So the only way we can guarantee a change is implemented globally without providing a custom controller to each and every step is to use a base wizard to always return our new base-*'s. I hope that clear things up